data-processing/entities/sentences/extractor.py

```diff
@@ -37,6 +41,86 @@
     "ᓸ",
 ]
 
+# regex patterns for detecting entities from the richer tex
+PATTERN_SECTION = r"\\(?:sub)*section[*]*\{[A-Za-z0-9 \{\}\\_.,:-]*\}"
+PATTERN_LABEL = r"\\label\{[A-Za-z0-9 \\_.,:-]*\}"
+PATTERN_ABSTRACT_BEGIN = r"\\begin\{abstract\}"
+PATTERN_ABSTRACT_END = r"\\end\{abstract\}"
+PATTERN_TABLE_BEGIN = r"\\begin\{table[*]*\}"
+PATTERN_TABLE_END = r"\\end\{table[*]*\}"
+PATTERN_FIGURE_BEGIN = r"\\begin\{figure[*]*\}"
+PATTERN_FIGURE_END = r"\\end\{figure[*]*\}"
+PATTERN_REF = r"\\ref\{[A-Za-z0-9 \\_.,:-]*\}"
+#TODO @dykang Fix to capture citations patterns like \cite[GRUs;][]{cho2004}
```

Comment:
Exactly!

Reply:
Sounds good! Maybe leave a note in the comments mentioning that the code can use the CitationExtractor here in the future in case higher accuracy is needed?